### PR TITLE
BETA: Register rakibahmed.is-a.dev

### PR DESCRIPTION
**Pull request by rafsaaanrakib**
Added `rakibahmed.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by rafsaaanrakib** [comment]
I just signed up for the beta registration at rakibahmed.is-a.dev. Can't wait to see what amazing projects or features are in store. Rakib, your work always delivers, and I'm eager to be a part of this journey. Let's beta-test the future together! 🚀💻 #BetaRegistration #Innovation"

File: domains/rakibahmed.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "rafsaaanrakib",
+        "email": "rakibyt13155@gmail.com"
+    },
+    "record": {
+        "CNAME": "rakibahmed"
```
Comment (DEV-DIBSTER) [inline domains/rakibahmed.json:7]: Invalid CNAME record.

**Review by DEV-DIBSTER — DISMISSED** [review]

**Comment by rafsaaanrakib** [comment]
how can i change the record?

File: domains/rakibahmed.json (hunk 1 above)
Comment (rafsaaanrakib) [reply domains/rakibahmed.json:7]: how can i change the record?

**Review by wdhdev — APPROVED** [review]

**Pull request merged**